reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -231,6 +233,50 @@ def save_checkpoint_and_serving_model(self, algorithm=None, env_string=None, con
         # To ensure SageMaker local mode works fine
         change_permissions_recursive(INTERMEDIATE_DIR, 0o777)
         change_permissions_recursive(MODEL_OUTPUT_DIR, 0o777)
+        
+    def set_up_checkpoint(self, config=None):
+        try:
+            checkpoint_dir = config['training']['restore']
+            print("Found checkpoint dir %s in user config." %checkpoint_dir)
+            return config
+        except KeyError:
+            pass
+
+        try:
+            # validate files in checkpoint channel and set restore dir in ray config
+            validation = 0
+            num_checkpoint_files = len(os.listdir(CHECKPOINT_DIR))
+                
+            if num_checkpoint_files is not (2 or 3):
```

Comment:
(2 or 3) will always resolve to 2. You can change to 
if num_checkpoint_files is not in [2,3]